src/nft/components/collection/UnavailableCollectionPage.tsx

```diff
@@ -0,0 +1,67 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import { darken } from 'polished'
+import { AlertTriangle } from 'react-feather'
+import { Link } from 'react-router-dom'
+import styled, { css, useTheme } from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const Container = styled(Column)`
+  height: 75vh;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 48px;
+  gap: 8px;
+`
+const LinkStyles = css`
+  color: ${({ theme }) => theme.accent1};
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => darken(0.1, theme.accent1)};
+  }
+`
+const InternalLink = styled(Link)`
+  ${LinkStyles};
+`
+const ExternalLink = styled.a`
+  ${LinkStyles};
+`
+export function UnavailableCollectionPage({ isBlocked }: { isBlocked?: boolean }) {
+  const theme = useTheme()
+  return isBlocked ? (
```

Comment:
for big ternary statements like these i prefer to make them early return style instead:

```
if (isBlocked) return (
    ...
 )

return ...
```

Reply:
Nice, I like that better and am changing it. Thanks for the suggestion!